src/ui/FKSelector.js

```diff
@@ -229,10 +229,10 @@ const FKSelector = fnObserver(props => {
 
     const {
         display,
-        query :
-        queryFromProps,
-        queryCondition :
-        queryConditionFromProps,
+        query: queryFromProps,
+        queryCondition: queryConditionFromProps,
+        rootType: rootTypeFromProps,
+        sourceName,
```

Comment:
Nach der Diskussion wegen den dynamischen Querys anzeigen gestern gefallen mir die Namen nicht für die neuen Attribute. Die sollten ähnlich heißen und so sprechend sein, damit man weiß dass sie zusammen gehören und was sie machen. alternateTypeSourceType und alternateTypeSourceAttribute oder so. Sind nur Vorschläge, aber ich hoffe ihr merkt in welche Richtung ich denke.